### DISCUSSION TITLE
Can I connect into a session that has been already generated?

### DISCUSSION THREAD

**Discussion by Joselguti** [SAP/PyRFC#287] [Q&A]
I am using another platform to generate sessions into my SAP Platform, however these connections don't get closed once I finish the code and there is no close object.

I wanted to know if it is possible to generate a connection in PyRFC with the parameters that were generated from this other connection in this other platform and close said connection with PyRFC.

The parameters I have access to:
![image (6)](https://user-images.githubusercontent.com/38333642/201937175-9da8939f-ec82-4521-be4e-6c8a14af8203.png)
![image(7)](https://user-images.githubusercontent.com/38333642/201937565-7ad532b1-256c-4c45-b7ee-228ef0e8838e.png)

Thank you in advance!

**Comment by bsrdjan** [comment]
Hello @Joselguti,

the sessions in question are long running RFC calls, from another platform, correct? What kind of platform it is and why closing sessions from that platform is not possible?

The party which created RFC connections should also close (clean-up) them, using RFC connection close() or cancel()method. Why external program (PyRFC) should close them?

CC: @mtolksdorf @Ulrich-Schmidt
